pr_watcher_notifier/views.py

```diff
@@ -72,8 +72,6 @@ def should_send_notification(data):
                     if fnmatch(modified_file.filename, pattern):
                         matched = True
                         matching_modified_files.append(modified_file.filename)
-                if matched:
```

Comment:
@nedbat, I had optimized this to do just the bare minimum hence the issue :smile: